cfme/tests/middleware/test_middleware_deployment.py

```diff
@@ -13,6 +16,10 @@
 pytest_generate_tests = testgen.generate(testgen.provider_by_type, ["hawkular"], scope="function")
 ITEMS_LIMIT = 5  # when we have big list, limit number of items to test
 
+RESOURCE_WAR_PATH = 'data/resources/middleware/cfme_test_war_middleware.war'
+RESOURCE_JAR_PATH = 'data/resources/middleware/cfme_test_jar_middleware.jar'
+RESOURCE_EAR_PATH = 'data/resources/middleware/cfme_test_ear_middleware.ear'
```

Comment:
I guess you want to use objects from `utils.path` to generate the paths, that will be more reliable.